etl/etl/utils.py

```diff
@@ -128,7 +128,7 @@ def parse_pare(pare_option):
     (PARE_FACTOR, KEYS_TO_INCLUDE) where PARE_FACTOR is an int
     by which proposals should be reduced, and KEYS_TO_INCLUDE
     is a list of proposals to include by key.  Only one of the
-    two will ever be returned."""
+    two will ever be a value, forcing the other to be None."""
```

Comment:
This still doesn't leave clear whether `(None, None)` can happen (I think it can, but the current language implies -- though doesn't quite outright state -- the opposite).  I think it's better to spend more words and lay out all three possible return states, so the caller is in no doubt.